app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -383,6 +366,14 @@ impl Colors {
 // === FRP ===
 // ===========
 
+/// The type of the [`selected`] FRP output of the component group.
```

Comment:
This doc comment does not explain anything. It explains where this thing is being used but not WHAT it represents. Please write all comments explaining semantics of the data structure (what is the purpose of its existence). Please apply to all doc comments in this and future PRs.

Reply:
Rephrased.